client/src/components/ElementEditor/Element.js

```diff
@@ -48,7 +48,7 @@ class Element extends Component {
    */
   handleKeyUp(event) {
     if (event.keyCode === 13) {
-      this.handleClick();
+      this.handleExpand();
```

Comment:
Nice catch, could be a PR on its own

Reply:
Could have been. But as it was effectively a no-op until this work was done, and it's such a minor, I feel it's OK. A sort of blurry line between this task and its dependency.

Reply:
This has been done now with #375